prepare/templates/rag/response_generation.py

```diff
@@ -16,7 +16,7 @@
 
 add_to_catalog(
     MultiReferenceTemplate(
-        instruction="Answer the question, basing your answer on the context",
+        instruction="Answer the question, basing your answer on the context\n\n",
```

Comment:
This is a non backward compatible change (in terms of results). I'm not sure why it's needed, it seems like more of an "format" issue them a template issue.  The format can place the instructions anywhere in the prompt.

Reply:
```suggestion
        instruction="Answer the question, basing your answer on the context",
```